### PR TITLE
use updated eyes-images version

### PR DESCRIPTION
**Pull request by jonathanadler**
Required according to Applitools:

https://help.applitools.com/hc/en-us/articles/360038739612?mkt_tok=eyJpIjoiTm1FNFpqZzBNamhsT1RBMyIsInQiOiJkWUtneVFRNkV5RTB4NHI2VmxhYUZacWtjNTdWc0hhbDg2SDNYOFgrTjJyek1VTDB5VFhZMGNRXC91VDBZblpcL2RTbGpuNk04cFFhY05VYzQzMnBlXC9Nc2cxV1pPTytJMkFTS1M1QUpOTUk3a0xWWEVubGJPcnV1ODBvZWQwUGs1OSJ9

### CODE REVIEW

**Comment by jonathanadler** [comment]
Tests are currently failing.
Do not merge

File: test/mocha.spec.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ describe('mocha', () => {
 
   eyes.it('should allow overriding default timeout', function () {
     expect(this.timeout()).to.equal(60000);
-    this.timeout(500);
-    expect(this.timeout()).to.equal(500);
+    this.timeout(5000);
```
Comment (MSiliunas) [inline test/mocha.spec.js]: why is the timeout increased?

Comment (jonathanadler) [reply test/mocha.spec.js]: Great question.
I think what happened is that the snapshot was taken and sent to the applitools servers.
and it took more than 500ms to finish and send back the result.
So the test failed.
I wasn't really sure how else to approach this.

**Comment by jonathanadler** [comment]
@MSiliunas pinging

**Pull request merged**